src/07-projects/projects.apib

```diff
@@ -111,6 +111,22 @@ Create a new project.
         + title: `New company website` (string, required)
         + description (string, optional)
         + starts_on: `2016-02-04` (string, required)
+        + billing_method (object, required)
```

Comment:
It is required in the new add flow though, it will not be possible to not choose a billing method. I can of course make it optional in the API, but then we should figure out how we are going to handle that, because the front-end will need that.

Reply:
I guess the frontend should make fields required __before__ even sending data to the api?